debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/SnapshotSourceIT.java

```diff
@@ -213,7 +213,7 @@ else if (Objects.equals(previousSnapshotSourceField, "last_in_data_collection"))
                     .isEqualTo(schemaEventsCount);
             assertThat(schemaChanges.ddlRecordsForDatabaseOrEmpty("").size()
                     + schemaChanges.ddlRecordsForDatabaseOrEmpty(OTHER_DATABASE.getDatabaseName()).size())
-                    .isEqualTo(useGlobalLock ? 1 : 5);
+                    .isEqualTo(1);
```

Comment:
@ani-sha Have you tried to investigate why the change is necessary? Why the test is not behaving in the same way as before?

Reply:
No @jpechane, I haven't checked that. Let me try it.